yb-voyager/cmd/import.go

```diff
@@ -110,6 +111,55 @@ func validateImportDataFlags() error {
 	if err != nil {
 		return err
 	}
+
+	return nil
+}
+
+var validCdcPartitioningStrategies = []string{"pk", "table", "auto"}
+
+func validateCdcPartitioningStrategyFlag(cmd *cobra.Command) error {
+	if importerRole != TARGET_DB_IMPORTER_ROLE {
+		return nil
+	}
+	if !changeStreamingIsEnabled(importType) {
+		if cmd.Flags().Changed("cdc-partitioning-strategy") {
+			utils.ErrExit("--cdc-partitioning-strategy is not supported for offline migration. Re-run the command without this flag.")
+		}
+		return nil
+	}
+	if cdcPartitioningStrategy == "" {
+		utils.ErrExit("cdc partitioning strategy is required")
+	}
+
+	if !lo.Contains(validCdcPartitioningStrategies, cdcPartitioningStrategy) {
+		utils.ErrExit("invalid cdc partitioning strategy: %s. Supported values are: %s", cdcPartitioningStrategy, strings.Join(validCdcPartitioningStrategies, ", "))
+	}
+
+	importDataStatus, err := metaDB.GetImportDataStatusRecord()
+	if err != nil {
+		return fmt.Errorf("error getting import data status record: %w", err)
+	}
+
+	if importDataStatus == nil || !importDataStatus.ImportDataStarted || bool(startClean) {
+		//if import data has not started or start-clean flag is used, allow the change in cdc partitioning strategy
+		return nil
+	}
+	if importDataStatus.CdcPartitioningStrategyConfig == "" {
```

Comment:
since we are marking it as a breaking release, let's just error out in this case?